src/options.js

```diff
@@ -31,6 +32,7 @@ export type Options = {|
   residual?: boolean,
   serialize?: boolean,
   strictlyMonotonicDateNow?: boolean,
+  errorHandler?: ErrorHandler,
```

Comment:
Design note: `Options` used to be a pure datastructure, with the intention that it's configured on a command-line, with no behavior attached to it. I can see how it is convenient to use it for this handler, but it's a bit weird.